src/common/util/src/os/win/win_shared_object_loader.cpp

```diff
@@ -150,11 +151,15 @@ std::shared_ptr<void> load_shared_object(const wchar_t* path) {
             return original;
         }();
         SetDllDirectoryW(dirname.c_str());
+        onednn_object = LoadLibraryW(L"C:/Users/shingyuk/work/openvino/build/src/plugins/intel_gpu/thirdparty/onednn_gpu_install/bin/onednn_gpu.dll");
```

Comment:
are you sure that will work on production env?

Reply:
I'm sorry to bother you. I missed to attach the `do_not_review` label.
This PR will not be merged into the master branch.

Reply:
don't worry, happy fixing ;)